pkg/config/viper.go

```diff
@@ -248,20 +258,15 @@ func (c *safeConfig) BindPFlag(key string, flag *pflag.Flag) error {
 	return c.Viper.BindPFlag(key, flag)
 }
 
-// GetConfigEnvVars returns a list of the non-sensitive env vars that the config supports
+// GetConfigEnvVars implements the DDConfigHelpers interface
 func (c *safeConfig) GetConfigEnvVars() []string {
 	return c.configEnvVars
 }
 
-// bindEnvAndSetDefault sets the default value for a config parameter, adds an env binding,
-// and tracks existing config env vars
-func (c *safeConfig) bindEnvAndSetDefault(key string, val interface{}) {
+// BindEnvAndSetDefault implements the DDConfigHelpers interface
+func (c *safeConfig) BindEnvAndSetDefault(key string, val interface{}) {
```

Comment:
This seems like purely a convenience method. Do we really need it?

Reply:
it's very useful (and we use it for most config options) so I'd say we need it yes